ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Problem:
```

Comment:
Yeah that's fair - it's trying to find a balance between a user being put off by so many questions and then not even filing versus requiring the necessary amount of information.

One thing is, if we err on the side of trying not to make the user feel overloaded with demands then at least we have the bug report and we can always ask for more information.